### PR TITLE
TCL30-65: Implement Header Search

### PR DESCRIPTION
**Pull request by alxmcr**
## Description

As user
I want to search place by name or address
so that I can find places around this place to search.

## Related Issue

[TCL30-65](https://the-collab-lab.atlassian.net/browse/TCL30-65)

## Acceptance Criteria

- You should be a search header in UI components.
- You should a search, settings, close, and help buttons.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

1. Checkout `ac-search-bar-header` branch.
2. Go to `/ui` to bottom page
3. You should be a header search.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ export function Button({
         </NavLink>
       ) : (
         <button
+          type={type}
```
Comment (jssckbl) [inline src/components/Button.js:24]: Type!

File: src/components/SearchForm.js [hunk 2]
```diff
@@ -1,20 +1,25 @@
-import { useContext, useState } from 'react';
+import { ReactComponent as SearchPlaceIcon } from '../assets/search-place-icon.svg';
+import { useContext } from 'react';
 import { GoogleMapsContext } from '../context/GoogleMapsContext';
 import { MapCenterContext } from '../context/MapCenterContext';
-import planetSearch from '../assets/search-location-solid.svg';
+import { Button } from './Button';
```
Comment (jssckbl) [inline src/components/SearchForm.js:5]: I love this Button component so much

**Comment by jssckbl** [comment]
This is great! I love how the search bar is working in conjunction with the other buttons along the row. Excited to see it implemented 😺

**Review by jssckbl — APPROVED** [review]

**Comment by alxmcr** [comment]
Yep! Jessica, I'm going to add this new component later, please! Maybe, all together 😄 . Thank you so much for your review.

**Pull request merged**